src/crate/crash/outputs.py

```diff
@@ -122,10 +124,13 @@ def json(self, result):
         yield self.to_json_str(obj)
 
     def csv(self, result):
-        wr = csv.writer(self.writer, doublequote=False, escapechar='\\')
+        wr = csv.writer(self.writer, doublequote=False, escapechar='\\', quotechar="'")
         wr.writerow(result.cols)
-        for row in result.rows:
-            wr.writerow(row)
+
+        json_dumps = lambda r: json.dumps(r, sort_keys = True)
```

Comment:
in the changes you write it is only required for python 2.x
then you could do a version check here